core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -131,11 +131,71 @@ describe('Exploration translation', function() {
     explorationEditorPage.saveChanges();
 
     explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
     explorationEditorTranslationTab.expectNumericalStatusToMatch(
       '(0/8)');
+
+    // click upload button
```

Comment:
Click the upload button.

Reply:
Removed the comment as the function name is self explanatory.